e3sm_to_cmip/__main__.py

```diff
@@ -722,10 +726,11 @@ def _setup_dirs_with_paths(self):
             copy_user_metadata(self.user_metadata, self.output_path)
 
         if not self.info_mode:
-            # Setup temp storage directory
             temp_path = os.environ.get("TMPDIR")
+
             if temp_path is None:
-                temp_path = f"{self.output_path}/tmp"
+                temp_path = f"{self.output_path}/tmp_{self.timestamp}"
```

Comment:
Making the `tmp` dir timestamp unique per `e3sm_to_cmip` invocation should prevent the issue where parallel `e3sm_to_cmip` jobs might delete the same shared `tmp` folder (resulting in silent CMORizing issues).